Library Catalog (COBISS).js

```diff
@@ -0,0 +1,521 @@
+					}
+				],
+				"date": "2023",
+				"url": "https://plus.cobiss.net/cobiss/si/sl/bib/139084803",
+				"attachments": [
+					{
+						"title": "Snapshot",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [
+					{
+						"tag": "dovoljene droge"
+					},
+					{
+						"tag": "nedovoljene droge"
+					},
+					{
+						"tag": "odpadne vode"
+					},
+					{
+						"tag": "čistilna naprava"
+					}
+				],
+				"notes": [
+					{
+						"note": "<p>Nasl. z nasl. zaslona</p>"
+					},
+					{
+						"note": "<p>Opis vira z dne 11. 1. 2023</p>"
+					},
+					{
+						"note": "<p>Bibliografija: str. 13</p>"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://plus.cobiss.net/cobiss/si/sl/bib/84534787",
+		"items": [
+			{
+				"itemType": "journalArticle",
+				"title": "Flood legislation and land policy framework of EU and non-EU countries in Southern Europe",
+				"creators": [
+					{
+						"lastName": "Kapović-Solomun",
+						"firstName": "Marijana",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Ferreira",
+						"firstName": "Carla S.S.",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Zupanc",
+						"firstName": "Vesna",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Ristić",
+						"firstName": "Ratko",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Drobnjak",
+						"firstName": "Aleksandar",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Kalantari",
+						"firstName": "Zahra",
+						"creatorType": "author"
+					}
+				],
+				"date": "2022",
+				"ISSN": "2049-1948",
+				"issue": "1",
+				"journalAbbreviation": "WIREs",
+				"libraryCatalog": "Library Catalog (COBISS)",
+				"pages": "1-14",
+				"publicationTitle": "WIREs",
+				"url": "https://plus.cobiss.net/cobiss/si/sl/bib/84534787",
+				"volume": "9",
+				"attachments": [
+					{
+						"title": "Full Text PDF",
+						"mimeType": "application/pdf"
+					},
+					{
+						"title": "Snapshot",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [
+					{
+						"tag": "EU legislation"
+					},
+					{
+						"tag": "Južna Evropa"
+					},
+					{
+						"tag": "Southern Europe"
+					},
+					{
+						"tag": "floods"
+					},
+					{
+						"tag": "land governance"
+					},
+					{
+						"tag": "policy framework"
+					},
+					{
+						"tag": "politika"
+					},
+					{
+						"tag": "poplave"
+					},
+					{
+						"tag": "upravljanje zemljišč"
+					},
+					{
+						"tag": "zakonodaja EU"
+					}
+				],
+				"notes": [
+					{
+						"note": "<p>Nasl. z nasl. zaslona</p>"
+					},
+					{
+						"note": "<p>Opis vira z dne 11. 11. 2021</p>"
+					},
+					{
+						"note": "<p>Bibliografija: str. 12-14</p>"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://plus.cobiss.net/cobiss/si/sl/bib/5815649",
+		"items": [
+			{
+				"itemType": "webpage",
```

Comment:
see my comment above - detecting "diplomsko" would work for the Slovenian catalog, but I'd need to also detect for all the other languages of COBISS catalogs

Reply:
This solution will work for doWeb, but I assume we wouldn't want to do this for detectWeb, since it's not an async function? Getting the English-language catalog page will take a little bit of time, and it seems like detectWeb just goes off the information on the page that's already loaded, i.e. the local language catalog page. Meaning, should I stick with my "guessing item type by icons" approximation for detectWeb?

Also, what's the recommended way for translators to read HTML from another page than the one you're on? I tried fetch, but it appears from this forum thread (https://groups.google.com/g/zotero-dev/c/O1TGIpfAdT0/m/dCZt08S2AwAJ) that maybe that isn't supported in Zotero 6? I'm currently using XMLHttpRequest(), but maybe there's a better/more modern way to do this. I also found the built-in function requestText(), but that seems to be primarily used for getting RIS, rather than HTML.